### DISCUSSION TITLE
Is it possible to latex like math notation in graphviz?

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#9736] [Q&A]
### Description

````qmd
---
title: "Graphviz with Math"
format: html
---

Using html for math symbols
```{dot}
//| echo: true
digraph G {

beta[label = <&#946;>] 
gamma [label = <&#947;>] 

beta -> gamma
}
```

Is there a way to use latex math notation?
```
digraph G {

beta[label = $\beta$] 
gamma [label = $\gamma$] 

beta -> gamma
}
```

````

**Comment by mcanouil** [comment]
Does GraphViz supports that? GraphViz, Mermaid, etc. are third party tools. What they can or not do is unrelated to Quarto.

**Reply by victorfeagins** [reply]
I don't think graphviz supports it natively. There are other tools that can manage this for example the latex packages dot2tex and graphviz.  dot2tex does have a python implementation. Curious if anyone has had any creative solutions for embedding such tools in documents.

**Reply by mcanouil** [reply]
If **GraphViz** doesn't inherently support mathematical equations, I recommend exploring external solutions related to GraphViz that can be integrated with **Quarto** through HTML and JavaScript.
Keep in mind that limitations in GraphViz and other third-party tools are independent of Quarto's capabilities. For a more tailored solution, consider researching online resources that specifically address GraphViz's mathematical rendering capabilities.
